bigtop-packages/src/rpm/impala/SPECS/impala.spec

```diff
@@ -76,8 +76,7 @@ Source7: statestored.svc
 Source8: catalogd.svc
 Source9: packaging_functions.sh
 Requires: bigtop-utils >= 0.7, /usr/sbin/useradd, /usr/sbin/usermod, openssl
-Requires: hadoop, hadoop-hdfs, hadoop-yarn, hadoop-mapreduce, hbase, hive >= 0.12.0+cdh5.1.0, zookeeper, hadoop-libhdfs, avro-libs, parquet, sentry >= 1.3.0+cdh5.1.0
-Requires: avro-libs, parquet, sentry
+Requires: hadoop, hadoop-hdfs, hadoop-yarn, hadoop-mapreduce, hbase, hive >= 1.2.1.2.3.0, zookeeper, hadoop-libhdfs
```

Comment:
Yes, we include Parquet, Avro, and Sentry in the Impala RPM rather than linking to another file. This is handled by the changes in `packaging_functions.sh`.